Source/Core/DolphinQt2/Config/Graphics/GraphicsSlider.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Dolphin Emulator Project
```

Comment:
None of these should be new widget classes. They don't expose any methods, signals, or properties beyond their parent classes.

They can be more simply implemented with something like:

```c++
QComboBox* MakeGraphicsChoice(const QStringList& options, const Config::ConfigInfo<int>& setting)
{
  auto* combo_box = new QComboBox();

  combo_box->addItems(options);
  connect(combo_box, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged),
          [&setting](int choice) { Config::SetBaseOrCurrent(setting, choice); });
  combo_box->setCurrentIndex(Config::Get(setting));

  if (Config::GetActiveLayerForConfig(setting) != Config::LayerType::Base)
  {
    QFont bf = combo_box->font();
    bf.setBold(true);
    combo_box->setFont(bf);
  }

  return combo_box;
}
```